src/Middleware/RequestThrottling/sample/Startup.cs

```diff
@@ -22,10 +22,10 @@ public class Startup
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddTailDropQueue((options) =>
+            services.AddStackQueue((options) =>
             {
                 options.MaxConcurrentRequests = Math.Max(1, _config.GetValue<int>("maxCores"));
-                options.RequestQueueLimit = Math.Max(1, _config.GetValue<int>("maxQueue"));
+                options.RequestQueueLimit = Math.Max(5, _config.GetValue<int>("maxQueue"));
```

Comment:
Why did you make this 5?

Reply:
This website is meant for people to run, and refresh the page with their fingers until they refresh too fast and it 503s. Setting the limit at 5 rather than 1 results in a more satisfying demo experience.

Reply:
Then add a config file to the sample and change it there. We shouldn't silently ignore what was configured.

Reply:
oh yeah I forgot about the Math.Max